packages/gatsby-plugin-gatsby-cloud/.babelrc

```
@@ -0,0 +1,3 @@
+{
+  "presets": [["babel-preset-gatsby-package", { "browser": true }]]
```

NIT: but that's not browser plugin :P
```suggestion
  "presets": [["babel-preset-gatsby-package"]]
```